### ISSUE TITLE
TTS Subtitles addon

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#5483]
### Why this would be helpful

Many projects do not have subtitles for the deaf or hard of hearing, making those projects inaccessible to those who can't hear or even just turned their volume down.

### How this addon works

This addon would be able to help in projects using the Text to Speech extension. It would show subtitles below the project player for anything the TTS voice says.

![A mockup of the subtitles area](https://user-images.githubusercontent.com/106490990/209415503-51221e41-5ff0-4e58-a1fb-a1d87ea051fd.png)

**Comment by retronbv** [comment]
correct me if im wrong but just piggybacking the execution of the tts block would be sufficient, read the param it is speaking, display it in the bottom part.. one concern would be xss in the tts inputs. should be easy enough to implement

**Comment by Secret-chest** [comment]
> correct me if im wrong but just piggybacking the execution of the tts block would be sufficient, read the param it is speaking, display it in the bottom part.. one concern would be xss in the tts inputs. should be easy enough to implement

Yeah, I think changing what the TTS block does by making it also set a JavaScript variable and that would be displayed in a box. Remember that the editor is actually loaded in the background, even when on the project page.

**Comment by illogicalapple** [comment]
Duplicate of #5514

**Comment by mybearworld** [comment]
[Here](https://scratch.mit.edu/discuss/topic/536596/)'s a suggestion (on Scratch) about this.

**Comment by WorldLanguages** [comment]
> [Here](https://scratch.mit.edu/discuss/topic/536596/)'s a suggestion (on Scratch) about this.

Wow, that's a lot of posts

**Comment by GrahamSH-LLK** [comment]
the easiest way to do this is to trap window.fetch and grabbing the text from there.
